assets/js/state/sapSystems.js

```diff
@@ -84,6 +84,20 @@ export const sapSystemsListSlice = createSlice({
         return sapSystem;
       });
     },
+    updateApplicationInstanceHost: (state, action) => {
```

Comment:
I think we can destruct here: `(state, { payload: {sap_system__id, old_host_id, _instance_number }})`.
We do it in some places and not in others though :sweat_smile:

Reply:
I think i will destructure only the payload since this way is cleared that we are comparing payload with an instance.